src/media.js

```diff
@@ -4,3 +4,34 @@ window.addEventListener('load', () => {
     activateCheckall('id[]', 'checkall');
     activateCloseSubmenus();
 });
+
+/**
+ * Drag and drop files anywhere to fill regular file input
+ */
+const fileInput = document.getElementById('file');
+const fileMenu = fileInput.closest('details');
+
+'drag dragstart dragend dragover dragenter dragleave drop'.split(' ').forEach(function(listener){
```

Comment:
I think I would prefer a simpler `for...of` construct. More like:

```js
const dragListeners = ['drag', 'dragstart', 'dragend', 'dragover', 'dragenter', 'dragleave', 'drop']
for (const listener of dragListeners) {
    // register listener
}
```

Reply:
Sure. Sorry for my js style :)